src/css/controls.less

```diff
@@ -0,0 +1,45 @@
+@import "shared-imports/mixins";
+
+// TO REMOVE
+@import "controls/imports/jwplayerelement";
+//@import "shared-imports/video";
+@import "controls/imports/display";
+@import "controls/imports/controlbar";
+@import "controls/imports/dock";
+//@import "shared-imports/title";
+@import "controls/imports/slider";
+
+//@import "shared-imports/captions";
+@import "controls/imports/rightclick";
+//@import "shared-imports/logo";
+@import "controls/imports/tooltip";
+@import "controls/imports/skipad";
+@import "controls/imports/cast";
+@import "controls/imports/nextup";
+@import "controls/imports/autostartmute";
+
+// State specific
+@import "controls/states";
+
+// flags
+@import "controls/flags/time-slider-above";
+@import "controls/flags/cast";
+@import "controls/flags/cast-available";
+@import "controls/flags/live";
+// Duplicate styles between controls-hidden/disabled
+@import "controls/flags/controls-hidden";    // To remove later
+@import "controls/flags/controls-disabled";
```

Comment:
These should no longer be needed. What do we need to think about keeping, replacing or testing after removal?